app/schemas.py

```diff
@@ -16,6 +23,22 @@ class UserBase(BaseModel):
     token: str
 
 
+class ProductCreate(BaseModel):
+    model_config = ConfigDict(from_attributes=True, arbitrary_types_allowed=True)
+
+    code: str
+
+
+class ProductBase(BaseModel):
+    id: int
+    off_source: ProductOFFSource | None
+    off_name: str | None
```

Comment:
`product_name` maybe? To follow off field names

Reply:
yeah you're right, i'll do the same for `off_product_quantity`